torchao/quantization/pt2e/lowering.py

```diff
@@ -12,21 +14,48 @@
     "lower_pt2e_quantized_to_x86",
 ]
 
+FUSION_PATH_REGISTERED = False
+
 
 def lower_pt2e_quantized_to_x86(
     model: torch.fx.GraphModule,
-    example_inputs: tuple[torch.Tensor, ...],
+    example_inputs: Optional[tuple[torch.Tensor, ...]] = None,
+    compile: bool = True,
+    **compile_options: Optional[dict],
 ) -> torch.fx.GraphModule:
     """Lower a PT2E-qantized model to x86 backend.
 
     Args:
     * `model` (torch.fx.GraphModule): a model quantized by PT2E quantization flow.
     * `example_inputs` (tuple[torch.Tensor, ...]): example inputs for the model.
+    * `compile` (bool): whether to torch.compile the model. Default is True.
+        Torch.compile brings more performance improvement.
+    * `compile_options` (dict): options for torch.compile.
 
     Return:
-    A GraphModule lowered to x86 backend.
+    A module lowered to x86 backend.
     """
 
+    if compile:
+        global FUSION_PATH_REGISTERED
+        if not FUSION_PATH_REGISTERED:
+            global torch
+            import torch._inductor.config
+
+            from torchao.prototype.inductor.fx_passes.quantization import (
+                _register_quantization_weight_pack_pass,
+                quant_lift_up,
+            )
+
+            torch._inductor.config.pre_grad_custom_pass = quant_lift_up
+            _register_quantization_weight_pack_pass()
+            FUSION_PATH_REGISTERED = True
```

Comment:
can this part happen during import of `x86_inductor_quantizer`?

Reply:
Thanks. I have changed per your suggestion.